include/wally_elements.h

```diff
@@ -162,6 +210,46 @@ WALLY_CORE_API int wally_asset_rangeproof(
     size_t len,
     size_t *written);
 
+/**
+ * Generate an explicit value rangeproof for a transaction output.
+ *
+ * The nonce for this function should be randomly generated.
+ * See `wally_asset_rangeproof_with_nonce`.
+ */
+WALLY_CORE_API int wally_explicit_rangeproof(
```

Comment:
args documentation is missing

Reply:
This is  a library convention to reduce header span, the API docs link to the `with_nonce` variant which documents a superset of the args taken.